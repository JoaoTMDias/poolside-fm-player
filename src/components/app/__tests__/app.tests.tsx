import React from "react";
import { shallow } from "enzyme";

import App from "../index";

describe("<App />", () => {
	it("should render a visualizer", () => {
		const component = shallow(<App />);

		expect(component).toMatchSnapshot();
	});
});
