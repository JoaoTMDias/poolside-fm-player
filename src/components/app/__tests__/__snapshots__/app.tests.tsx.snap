// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render a visualizer 1`] = `ShallowWrapper {}`;
